toqito/matrices/hadamard.py

```diff
@@ -20,28 +20,25 @@ def hadamard(n_param: int = 1) -> np.ndarray:
 
     .. math::
         \left( H_n \right)_{i, j} = \frac{1}{2^{\frac{n}{2}}}
-        \left(-1\right)^{i \dot j}
+        \left(-1\right)^{i \cdot j}
 
     Examples
     ==========
 
-    The standard 2-qubit Hadamard matrix can be generated in :code:`toqito` as
+    The standard 1-qubit Hadamard matrix can be generated in :code:`|toqito⟩` as
 
-    >>> from toqito.matrices import hadamard
-    >>> hadamard(1)
-    array([[ 0.70710678,  0.70710678],
-           [ 0.70710678, -0.70710678]])
+    .. jupyter-execute::
 
-    References
-    ==========
-    .. bibliography::
-        :filter: docname in docnames
+     from toqito.matrices import hadamard
 
+     hadamard(1)
 
+    References
+    ==========
+    .. footbibliography::
```

Comment:
I think most of the changes in this PR are not needed. The main branch already has these changes....

In the future, if you create a PR, please make sure your main branch is up to date....